config-test/test_server_details.py

```diff
@@ -11,10 +12,9 @@ def conf():
 
 
 def test_version(conf, env):
-    response = requests.get(conf.get(env, 'reader_server') + '/__version__')
+    response = requests.get(conf.get(env, 'reader_server') + '__version__')
```

Comment:
maybe we can use `rstrip('/')` so that we handle both the case when ``reader_server`` contains a trailing slash or when it does not.